recipes/r-sahu/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set version = '0.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sahu
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/sk-sahu/sahu
```

Comment:
can you please use a tarball?

Reply:
Hi @bgruening I added the tarbal link in url. In this case its failing for windows, Tried to check log but couldn't figure out. With git_url it was passing all platforms. Should I do [this skipping platform](https://github.com/conda-forge/staged-recipes#3-how-do-i-exclude-a-platform)